### DISCUSSION TITLE
dbt power user visual studio code extension : features stopped working

### DISCUSSION THREAD

**Discussion by vidya-sva** [AltimateAI/vscode-dbt-power-user#531] [Q&A]
I recently installed the dbt power user extension and followed the configuration instructions, which included adding

"files.associations": {
        "*.sql": "jinja-sql"
    }
, and making the Python interpreter selection always visible. It used to function perfectly, and I could view model graph visualisation, etc., but it no longer does. I haven't made any changes. I tried reinstalling the extension, clearing cache, restarting, and experimenting with different settings, but nothing worked. It appears that the extension is not working because I see it cannot detect ref function or source definition, and there is no model graph or visualisation. I am unable to generate models from source definitions or run models via the buttons. Any assistance is greatly appreciated.
<img width="1019" alt="dbt_power_extension_issue" src="https://github.com/innoverio/vscode-dbt-power-user/assets/30149223/d2c53570-b671-4489-9d53-9250b3af7238">

**Comment by anandgupta42** [comment]
@vidya-sva Can you let us know the version of the extension that you are using?

**Reply by anandgupta42** [reply]
@vidya-sva  There is a known bug when the model visualization gets pushed below the legends.  Can you try increasing the size of the pane or trying scrolling down?

**Reply by anandgupta42** [reply]
@vidya-sva Did you manage to resolve the issue?

**Reply by vidya-sva** [reply]
@anandgupta42  Apologies for the delayed response, I was busy with a different piece of work. I've got an update to share. It's not related to making adjustments to legends. I went ahead and reinstalled the extension, and now it's working. It's a bit odd because I had previously attempted all these steps, but it might be because of the version. I'm currently using version 0.17.5, whereas the last time I tried this, I believe I was on version 0.16.0. However, I'm not entirely certain about the version number, but it was the latest one available at that time.

**Comment by anandgupta42** [comment]
Awesome. I am glad you got it working.
